src/main/java/edu/wisc/my/rssToJson/controller/RssToJsonController.java

```diff
@@ -26,6 +30,37 @@
     public void setRSSToJSONService(RssToJsonService rssToJsonService) {
         this.rssToJsonService = rssToJsonService;
     }
+
+    @RequestMapping(value="/rssTransform/{feed}/xml")
```

Comment:
this url seems a little backwards, can we have it be `/xmlTransform/{feed}`?

Reply:
We could. Since the system is designed to be flexible enough to incorporate any kind of content - not just xml - maybe it should be a more generic name.